docs/molgenis/dev_quickstart.md

```diff
@@ -134,6 +134,12 @@ Go to IntelliJ Preferences -> Build tools -> Gradle and then set Run tests using
 
 See https://linked2ev.github.io/devsub/2019/09/30/Intellij-junit4-gradle-issue/
 
+If you find that tests take long, then in 'edit configuration' of your test runner and in  'build and run' section switch from 'All in package' to 'tags' 
+and set to '!slow'  
```

Comment:
suggest more direct / active tone for docs; ( 'skip test test marked as 'slow'......

Reply:
indeed, I rewrote it